Makefile

```diff
@@ -1,12 +1,23 @@
+.DEFAULT_GOAL 					:= all
```

Comment:
The indentation for these variable assignments are a bit all over the place, can you align them, please?

Reply:
Resolved. Indentation fixed.